packages/gitbook/src/lib/pages.ts

```diff
@@ -1,53 +1,99 @@
+import type { GitBookLinker } from '@/lib/links';
+import { joinPath } from '@/lib/paths';
+import type { FlatPageEntry } from '@/lib/sitemap';
 import {
     type Revision,
     type RevisionPage,
     type RevisionPageDocument,
     type RevisionPageGroup,
     RevisionPageType,
 } from '@gitbook/api';
+import type { ListItem, Paragraph, RootContent } from 'mdast';
 
 export type AncestorRevisionPage = RevisionPageDocument | RevisionPageGroup;
 
+interface ResolvePageOptions {
+    /**
+     * Whether to include page groups in the results.
+     */
+    includePageGroup?: boolean;
```

Comment:
The naming is a bit confusing, as I first thought you meant "in the ancestors".
```suggestion
    /**
     * Whether to return a page group if it matches the path. If not it'll resolve to the first document in it.
     * @default false
     */
    returnPageGroup?: boolean;
```